src/ui/SmartSnippet/UserFeedbackBanner.ts

```diff
@@ -78,7 +84,7 @@ export class UserFeedbackBanner {
   }
 
   private buildLabel() {
-    return $$('span', { className: LABEL_CLASSNAME }, l('UsefulnessFeedbackRequest')).el;
+    return $$('span', { className: LABEL_CLASSNAME, id: this.labelId }, l('UsefulnessFeedbackRequest')).el;
```

Comment:
Could we use a better semantic tag than `span` ?

Reply:
On its own, the semantic choice would probably be a `label`. However, a `label` cannot be associated with multiple controls. I considered using a `fieldset` and a `legend` back when I considered making those buttons `input[type="radio"]`, but I let go of the idea at the same time that I chose to make them toggleable buttons.

I'll keep them as toggleable buttons but I'll make this a `legend` and introduce a `fieldset`.